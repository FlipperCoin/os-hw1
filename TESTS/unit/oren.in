sleep 100&
sleep 200&
showpid > mypid
jobs > myjobs
pwd > mypwd
cat mypid
cat myjobs
cat mypwd
showpid | showpid
jobs | jobs
pwd | pwd
jobs | showpid
quit kill