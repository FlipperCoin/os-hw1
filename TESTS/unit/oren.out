smash> smash> smash> smash> smash> smash> smash pid is 24020
smash> [1] sleep 100& : 24026 0 secs
[2] sleep 200& : 24027 0 secs
smash> /tmp/smash_test
smash> smash pid is 24020
smash> smash> /tmp/smash_test
smash> smash pid is 24020
smash> 